### PR TITLE
fix: disabled flipper button clicks

### PR DESCRIPTION
**Pull request by eljefe223**
# Pull Request

## 📖 Description
Adds `pointer-events: none` to prevent clicks. 

Other option would be to place a `click` handler on the template and `stopPropagation`. But css approach seems less intrusive.

```ts
public clickHandler(e: Event & MouseEvent) {
        if(this.disabled) {
            e.stopImmediatePropagation();
        }
    }
```

### 🎫 Issues
Closes: #5385 

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

**Review by scomea — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
I think we have this same problem on other components as well (e.g. button itself). Could we do a quick survey of the components and see if there are other places this fix is needed, and explore what our standard approach should be?

**Comment by scomea** [comment]
I am wondering if this type of "functional" style might best be implemented by setting the attribute from the code side like when the disable prop is toggled, vs. in a style sheet.  Would avoid folks needing to update style sheets for this kind of fix.  Down side is that it would not be easy to override.

**Comment by eljefe223** [comment]
> I am wondering if this type of "functional" style might best be implemented by setting the attribute from the code side like when the disable prop is toggled, vs. in a style sheet. Would avoid folks needing to update style sheets for this kind of fix. Down side is that it would not be easy to override.

@scomea Open to others feedback. We could also make it a style sheet behavior that gets added to the components that need it.

**Comment by chrisdholt** [comment]
> I think we have this same problem on other components as well (e.g. button itself). Could we do a quick survey of the components and see if there are other places this fix is needed, and explore what our standard approach should be?

This is what Jeff is focused on - he’s just taking them individually.

**Review by EisenbergEffect — APPROVED** [review]

**Comment by chrisdholt** [comment]
> > I am wondering if this type of "functional" style might best be implemented by setting the attribute from the code side like when the disable prop is toggled, vs. in a style sheet. Would avoid folks needing to update style sheets for this kind of fix. Down side is that it would not be easy to override.
> 
> @scomea Open to others feedback. We could also make it a style sheet behavior that gets added to the components that need it.

If we wanted to handle this behaviorally without styles we can just check before emitting or firing events. I'm open on how we approach this, but pointer-events will fix the current bug. We can definitely consider a larger change as a feature request though.

**Comment by scomea** [comment]
> If we wanted to handle this behaviorally without styles we can just check before emitting or firing events.

I don't think so, click is native so we don't actually emit it.  We could try and stopPropagation on it.

**Comment by chrisdholt** [comment]
> > If we wanted to handle this behaviorally without styles we can just check before emitting or firing events.
> 
> I don't think so, click is native so we don't actually emit it. We could try and stopPropagation on it.

For flipper I don't think the click is native...it doesn't exist, so that is being handled elsewhere. It's the case for key events but it's not actually a native element for this control: https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/flipper/flipper.template.ts

**Review by bheston — APPROVED** [review]

File: change/@microsoft-fast-components-ce85fe24-cfca-4d69-a8e5-5a4ba3305d77.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: diasbled flipper button clicks",
```
Comment (janechu) [inline change/@microsoft-fast-components-ce85fe24-cfca-4d69-a8e5-5a4ba3305d77.json:3]:
```suggestion
  "comment": "fix: disabled flipper button clicks",
```

**Review by janechu — APPROVED** [review]

**Pull request merged**